build/lib/android/startup.ejs

```diff
@@ -0,0 +1,3 @@
+this._startup = function (global) {
+<%- script %>
+}
```

Comment:
First, I think what you're doing via the "startup.ejs" is a nice elegant solution for "capturing" a snapshot without running it. I like it.

I do think we should delete/undefine the startup function when invoked to remove it from the global namespace. This will work fine for LiveView since it terminates/restarts the JS runtime when a file changes. And perhaps we should give the startup function a more meaningful name to make it more obvious what we're doing in the `TiLaunchActivity.java` side. What do you think?

```javascript
global.startSnapshot = function (global) {
	delete global.startSnapshot;
<%- script %>
};
```

Oh and is passing the "global" object as an argument necessary?